examples/reactnative/android/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,7 @@
       </activity>
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
       <meta-data android:name="com.bugsnag.android.API_KEY"
-                 android:value="8f4b9d17b582d6e4c74d43d3e295b4c5"/>
+                 android:value="5d1ec8bd39a74caa1267142706a7fb20"/>
```

Comment:
Was this deliberate and/or does it matter?

Reply:
This is not deliberate and doesn't particularly matter - I'll revert to the last value regardless.